web/auth.go

```diff
@@ -527,8 +527,14 @@ func (h *loginHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	})
 
 	if r.FormValue("backto") != "" {
-		session.Data["backto"] = r.FormValue("backto")
+		backto := r.FormValue("backto")
+
+		_, err := url.ParseRequestURI(backto)
+		if err == nil {
+			session.Data["backto"] = backto
+		}
```

Comment:
from dms

> My theory is any external url would be valid so we shouldn’t allow that, only set backto when we detect an invalid url.

so should this be `err != nil`?

Reply:
i tested in the temp env from the linked app pr and still run into the same issue described into the ticket: 

https://pr-6610-appmain-bplesliplq-uc.a.run.app/login?backto=https://www.sumitsahoo.com/

Reply:
Sorry yes, `!=` I switched the logic around and forgot to swap the operator, thanks

Reply:
ok confirmed in test env that it doesnt work anymore. have been trying to find places where we do leverage `backto` to make sure it doesnt break but none of them seem to be working. now confirming that this is existing behavior

Reply:
@dgottlieb posted in app-chat earlier that he noticed he wasn't being redirected back to the right place post-login: https://viaminc.slack.com/archives/C02KFSY4H89/p1729521583147719